apps/Gruntfile.js

```diff
@@ -138,6 +138,17 @@ config.copy = {
     files: [
       {
         expand: true,
+        cwd: 'lib/blockly',
+        src: ['??_??.js'],
```

Comment:
Neat! Hadn't seen the glob `?` before.

Fun fact after [looking it up](http://en.wikipedia.org/wiki/Glob_%28programming%29#Syntax), in Windows cmd.exe, `?` means 0 or 1 characters, while in unix/powershell, `?` is exactly 1